cw_4/src/Pages/Payment/Payment.jsx

```diff
@@ -1,135 +0,0 @@
0Oks0cZ6HPa0+BN156faahebMqoSejlGjwuVz7Q4ZPUPIhifK7i4tBJ7XOPT1r11uzFdEM0lNIG85y5gPy3VnwNa2c9/wVvHTe6ht+ToxrgdQbhfq6jW7yl5PD4Bwu3N3uN18t4uNmlo3ljrSSfZsPOCeLh2C6oKtufQvvoXu80h1s38uIwNJa6WNpHEF7QR3Er6U9XG++R7X245XB1u2y5ic0c+p6TqSekk8SrC3M1gbUzQ+0jDgOtp19xVy7A7uDnrx7FSnN7yajpyXGiwFlUC+EREAREQBERAEREAREQBERAEREBhUHvNqM+ITfwBrPAf5V+Fc37QymasncNS+ZzQOk5sgHfZX+zo/Ub+yKGe/Il7ll0m3VFQ00FOM0r2RMDhGBYEgEjMdL6qQbM7aUtcSyMlsgF+Tfo63SOYqFxbqHchmdORPa+UNHJg280nj3qBYRVPgqIpG3DmSt/8Aazh4XC7WPTbGXdvdHDyLq2utbMuCp3m0EbnMcJg5ji1w5M6EGxCl1FVsljZKw3a9ocD1EXVLb1cK5Gs5QCzZ2h4/nGjh8D3lb/ZLbBtPhT3O1khcY2N9Yu1YOwX8AorMaLqjOv1/8/ckhkSVkoT9CUY5t5RUsxhkLy8AE5G5gL8x61ttn8ajrIuWja8MJIBe3Le3EgdConAMKmxCrDC4kvcXyyc4bfyj2ngP8LoKjpWRMbHG0NYwBrWjgABYBc5NNdSUVvL1O8e2drcn/SRveZh3LUEluMdpB+Hj7lVuyGPmF7Y3m8ZcC0n0XX5jzA9CviqgEjHMdwc0tPeLLmivpTFLJE7Qse5h7iQvaaK8qmVFi917P7r9Di+2WPbG2Hw/f2OhsexZtPSSVB9Flx1uIs0eK50ke5ziTq5xJPSXOPzKmG1G05nw+ip813Zc0v8A+ZLGA9pBd3DpXm3c4N9JrWEi7IftX34XHmDvPwVrEh4eqUpc/wAfyVsiff2RjHj+S2ti8IFHRxxnzrcpIf4zqfDh3Kh8UqOUnlkvfNI91/xG3uXQu01XyNJPJ6sTz32K5tvYdy87O1lKc2dZ+kYxgi8d0lLkw9rueSSR/vyg+DQqm2ppnx1c7HizuUce0ONwfCyvbZGk5Kip4+dsTL9trrxbWbIU9e27/IkaLNlb53YfWHUVXpyVC6UnwyxbjudUVHlFabv9tGUOaKVl43uzZ2+c12g1HpBXBhWL09SzPBI17eo6jtHEKjtp9jaqh8p4D4r25RnAXNhmHo34LVYLi01JK2aFxBaQSPRe3nY4c4I8OKtW4sL13lb3K1WROl9E1sdKtCo7eljX0isMTTdkHkacC/0/Dh4q0se2hbBQOqxxMYMYPO948geJ17CufSXOOpu5xuSTxcTqSe03UPZ9WsnN+hLn2+VQXqb6mwAnDpawjVsrWN6MnBx8SFjYOt5HEKd3MX8mex4y/Gysepmw9uGOom1ERIhLfPGrwLk9pcqbilLS144tIcO0WP7K3VN3Rmn93oVbYqqUGjqILK8uHVIlijkHB7Gu8QCvUsM2wiIgCIiAIiIAiIgCIiAIiIAiIgPNXShkb3ng1rneAJXNVBWOZLHPYFzZGy68CQ4OIXSGM0JnglhzZeUjdHmAvbMCL271BKHdPA115J3SNyuGXKG2JGjgRzhXcS6utS6/UpZdNljj0+h+sV3o0vIEwB5mc0gNc0gNcRxJOht1cVXuxmDuq6yJliWteJJHdDWnMbnpJFlMfqjkz/8AlDJ05PLt42up7s1s3T0MeSEEk6ue7V7j1no6lJ31NMGqt2yPubrZp28I0e9fCeWojIB5UDhJ+Hg/3G/cqRJ0/wC/BXLvV2j5GD6LGftJgQ7+GLge88PFRHdfsz9Jn+kSD7KE3APB0nEdzePbZS4k+6ocpca7EWVBWXKMefUnu7nZr6HThzxaaWzn34tHos7h71MEWVlzm5ycn6mpCChHpRgqid6eHcjiD3AeTM1so6M3mPHi2/4lexUX2z2PZiHJEyGN0eaxADrh1rjXrAU2Jcq7FJ8EOVU7K9FyUErv3VYLyFHyrh5dQeUPSGcGDw171qI90kYcCalxAIJGRouL6i/WrJhYGgNAsAAAOoaKzmZUbIqMPyVsTGlCTlP8ES3rVWTDpBzyOZH4m59wKpSgg5SWNnrPa3xIV87Z7M/T42RmUxhj8+gBubWHHtUfwjdeyCeOb6Q5/JuDspY0Xt1pjZFdVTT53/Y9yaLLLU0ttjc7d41JQUbXw5c+eONocLi3OLfygqM0O9ttvtqZ1+mNwI8HWspltZszFXxNjkc9mR2dpYR51i3UEaixKhE+6J1/IqtP4mfIqKh47hpZyd3LIUta+DwbYbxBV076eKEsbJYOc8gmwINgBz6cVCsMw+SolZBELvkNh1DnceoBWNSbotftKk26GMsfFxKm+z2zFLRNIgZZx857vKee08w6grHiaaYONRD4e62alYVvvTxIAw0MZu2BjS7rdbK2/dr3qHYVg1TUlwp4nSFvnZbaX4cSFZmIbr3TSvmfVkukcXH7Mc/AceAFh3KTbG7LMw+N7A/O57sxcQBzWAsiyq6qkobsPFnZbrPZFPf6IxP7pJ4s/wCS0lXTPie6ORpa9pyuaeII5l08VAdpN27aqpkqBOY+UtdoYDqABe/clPaDcvqcHl2BpH6e7NruzreVw+HW5jBjP4TYe6ylSjmxmzRoI3xcqZA52YXaG20sR7lI1n2uLm3HjU0Kk1BKXJlERRkgREQBERAEREAREQBERAEREAREQBeauqOTjc/KXZWk5Wi7iRwAHSvSsWQFBV+C4pWVJkkp5GuleBdw8lgvYDjwAV14FhMdLAyCMaMFr85PO49ZK2OVZU92Q7ElpokV6ceNbb5bAWURQFgIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgCIiAIiIAiIgP/Z" alt="" />
```

Comment:
The image source is too big. Put it into some other file and import it from there.